src/components/Accounts/ViewFeesFines/ViewFeesFines.js

```diff
@@ -153,7 +155,7 @@ class ViewFeesFines extends React.Component {
         <Row>
           <Col>{t}</Col>
           {(n > 0) ?
-            <Col style={{ marginLeft: '5px' }}>
+            <Col className={css.popoverWrap}>
```

Comment:
Would it make more sense to just use Layout here instead to maintain uniformity?

```
<Col>
  <Layout className="margin-start-gutter">
    <Popover>...</Popover>
  </Layout>
</Col>
```